.github/workflows/scheduled_test.yml

```diff
@@ -0,0 +1,49 @@
+name: Scheduled Test
+
+on:
+  push:
+    branches: [develop]
+  pull_request:
+#  schedule:
+#    - cron: '0 8 1 * *'
+#  repository_dispatch:
+#    types: [ custom-event ]
+
+jobs:
+  scheduled_test:
+    runs-on: ${{ matrix.os}}
+    strategy:
+      matrix:
+        os: [ windows-latest, ubuntu-latest ]
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python 3.12
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
+      - name: Install dependencies with develop aequilibrae
```

Comment:
@pedrocamargo this is one of the workflows I told you wasn't working. I probably must have done something stupid when setting the installation of AequilibraE and its dependencies above. For both tests (on Windows and Linux), the spatialite dependency seems not to be working, although it uses the same configuration as the testing workflow.

When we finish this workflow setup, I'll try to add the `repository_dispatch` to it, so we have only one workflow to run when AequilibraE is modified or periodically (i.e. remove lines 4-6 and uncomment lines 7-10).